docs/reference/commandline/system_prune.md

```diff
@@ -0,0 +1,70 @@
+<!--[metadata]>
++++
+title = "system prune"
+description = "Remove unused data"
+keywords = [system, prune, delete, remove]
+[menu.main]
+parent = "smn_cli"
++++
+<![end-metadata]-->
+
+# system prune
+
+```markdown
+Usage:	docker system prune COMMAND
+
+Delete unused data
+
+Options:
+  -a, --all     Remove all unused images not just dangling ones
+  -f, --force   Do not prompt for confirmation
+      --help    Print usage
+```
+
+Remove all unused containers, volumes and images (both dangling and unreferenced).
+
+Example output:
+
+```bash
+$ docker system prune -f -a
```

Comment:
Looks like the command doesn't match the output (if `-f` is provided, there should not be a prompt for confirmation?)